classes/local/lti_helper.php

```diff
@@ -188,4 +192,27 @@ public static function get_filter_lti_launch_url(int $ocinstanceid, int $coursei
         }
         return $ltilaunchurl;
     }
+
+    /**
+     *  Calls the URL of the presention node of opencast.
+     *
+     *  This function calls an api endpoint because the url of the engage.ui is different depending on
+     *  the installation (All-In-One or Multiple Servers).
+     *
+     * @param int $ocinstancid
+     * @return string $url the url of the engage.ui of the opencast installation
+     * @throws \moodle_exception
+     */
+    public static function get_engage_url(int $ocinstancid) {
+        $api = api::get_instance($ocinstancid);
+        // Endpoint to call the engage.ui url of presentation node.
+        // Make sure the api user has the rights to call that api endpoint.
+        $engageurlendpoint = '/api/info/organization/properties/engageuiurl';
+        $result = json_decode($api->oc_get($engageurlendpoint), true);
```

Comment:
Please use opencastapi (Opencast PHP Library) property: [getOrgEngageUIUrl](https://github.com/elan-ev/opencast-php-library/wiki/OcBaseApi#getorgengageuiurl)
So that would be something like this:
`$api->opencastApi->baseApi->getOrgEngageUIUrl();`

`api->oc_get` is no more recommened and it will be removed in near future!